standards/factories.py

```diff
@@ -0,0 +1,34 @@
+from factory import Sequence, SubFactory
+from factory.django import DjangoModelFactory
+
+from standards.models import Standard, GradeBand, Category, Framework
+
+class StandardFactory(DjangoModelFactory):
+    class Meta:
+        model = Standard
+
+    name = Sequence(lambda n: "Standard %03d" % n)
+    gradeband = SubFactory('standards.factories.GradeBandFactory')
+    category = SubFactory('standards.factories.CategoryFactory')
+    framework = SubFactory('standards.factories.FrameworkFactory')
+
+class GradeBandFactory(DjangoModelFactory):
+    class Meta:
+        model = GradeBand
+
+    name = Sequence(lambda n: "GradeBand %03d" % n)
+
+class CategoryFactory(DjangoModelFactory):
+    class Meta:
+        model = Category
+
+    name = Sequence(lambda n: "Category %03d" % n)
+    framework = SubFactory('standards.factories.FrameworkFactory')
+    # Need to set parent to none on subfactory so that we don't end up with infinite recursion
+    parent = SubFactory('standards.factories.CategoryFactory', parent=None)
```

Comment:
nice tactic. not sure if we'll always need a category to have a parent, but if we find that we're making lots of CategoryFactory calls with parent=null then we could always refactor this later to have no parent be the default.